recipes/matplotlib-corefonts/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "matplotlib-corefonts" %}
```

Comment:
Actually, under [section 2](https://github.com/conda-forge/staged-recipes/pull/8497/files#diff-13de976cfe10a42b2d3a376d8d8d3b2dR23) of the license, it is not allowed to rename this project.  I think we must call this `corefonts` and we cannot rebrand this as part of matplotlib.

Reply:
Or maybe even better, call it `mscorefonts`